fig2_validation.py

```diff
@@ -46,212 +44,204 @@ def main(   par="./params_dMSN.json",        \
+
+            if sec.name().find('axon') < 0:
+
+
                 # these channels are not in the axon section
                 h.setpointer(pointer, 'pka', seg.kaf )
                 h.setpointer(pointer, 'pka', seg.cal12 )
                 h.setpointer(pointer, 'pka', seg.cal13 )
                 h.setpointer(pointer, 'pka', seg.kir )
-                
+
                 if sec.name().find('soma') >= 0:
-                    
-                    
+
+
                     # N-type Ca (can) is only distributed to the soma section
                     h.setpointer(pointer, 'pka', seg.can )
-                    
-            
-    
+
+
+
     # configure simulation to record from both calcium pools.
-    # the concentration is here summed, instead of averaged. 
+    # the concentration is here summed, instead of averaged.
     # This doesn't matter for the validation fig, since relative concentration is reported.
     # For Fig 5B, where concentration is reported, this is fixed when plotting.
     # -> see the plot_Ca_updated function in plot_functions.
     if sim == 'ca':
-        
+
         print('configure', sim, 'simulation')
-        
+
         for i,sec in enumerate(h.allsec()):
-            
+
             if sec.name().find('axon') < 0: # don't record in axon
-            
+
                 for j,seg in enumerate(sec):
-                    
+
                     sName = sec.name().split('[')[0]
-                    
-                    
+
+
                     # N, P/Q, R Ca pool
                     cmd = 'ca_%s%s_%s = h.Vector()' % (sName, str(i), str(j))
                     exec(cmd)
                     cmd = 'ca_%s%s_%s.record(seg._ref_cai)' % (sName, str(i), str(j))
-                    exec(cmd)   
-                    
+                    exec(cmd)
+
                     # the L-type Ca
                     cmd = 'cal_%s%s_%s = h.Vector()' % (sName, str(i), str(j))
                     exec(cmd)
                     cmd = 'cal_%s%s_%s.record(seg._ref_cali)' % (sName, str(i), str(j))
-                    exec(cmd)   
-                    
-                    
+                    exec(cmd)
+
+
                     # uncomment here if testing kaf blocking effect on bAP
                     #block_fraction = 0.2
                     #gbar           = seg.kaf.gbar
                     #seg.kaf.gbar   = (1 - block_fraction) * gbar
-    
-    
-              
-    # solver------------------------------------------------------------------------------            
+
+
+
+    # solver------------------------------------------------------------------------------
     cvode = h.CVode()
-    
+
     h.finitialize(cell.v_init)
-    
+
     # run simulation
     while h.t < tstop:
-                
+
         h.fadvance()
-        
-    
+
+
     # save output ------------------------------------------------------------------------
-    
-    if sim == 'ca':
+
+    for name in ["Results/Ca", "Results/FI", "Figures"]:
+    try:

```

Comment:
This is a syntax error; the try/except must be indented under the "for"

Reply:
True. Should I cancel this commit or you just need the file corrected?